### ISSUE TITLE
Main container can not `depends_on` logging container(firelens_log_router)

### ISSUE THREAD

**Issue by rluisr** [aws/copilot-cli#3053]
I would like to depend on `firelens_log_router` on the main container. But current copilot does not support this.
```
$ copilot deploy
✘ execute svc deploy: validate manifest against environment sbx: validate container dependencies: container firelens_log_router does not exist
```

manifest
--------
```yml
image:
  depends_on:
    firelens_log_router: HEALTHY # I think `logging stanza` does not support `healthcehck` so please ignore `HEALTHY`.

logging:
  ...
```

**Comment by iamhopaul123** [comment]
Hello @rluisr. This request makes a lot of sense. Will send out a PR soon to address this. Thank you for reporting this issue!

**Comment by efekarakus** [comment]
This is now released 🎉 in v1.13.0 https://github.com/aws/copilot-cli/releases/tag/v1.13.0 !
